src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Components/LongRunningOperationHelper.cs

```diff
@@ -77,12 +77,12 @@ internal LongRunningOperationHelper(string activityName, Func<ResourceManagerRes
         /// Waits for the operation to complete.
         /// </summary>
         /// <param name="operationResult">The operation result.</param>
-        internal string WaitOnOperation(OperationResult operationResult)
+        internal string WaitOnOperation(OperationResult operationResult, string resourceId = null, string apiVersion = null)
```

Comment:
We should make this change for the Remove appliance cmdlet too

Reply:
done

Reply:
Done